src/BcpModeler.cpp

```diff
@@ -532,6 +540,12 @@ void BcpLpModel::generate_vars_in_lp(const BCP_lp_result& lpres,
 	}
 
 
+// 	static int  cpt = 0;
```

Comment:
Delete? Or uncomment with conditional compilation?

Reply:
I don't know where he comes from ... Change of tab ...